static/css/index.css

```diff
@@ -0,0 +1,28 @@
+@keyframes gradientAnimation {
+  0% {
+    background-position: 0% 50%;
+  }
+  100% {
+    background-position: 100% 50%;
+  }
+}
+
+.gradient-flow {
+  background: linear-gradient(to right, #ff5a00, magenta, cyan);
```

Comment:
Please have a look at the colors that are used for the current gradients and reuse them, please

Reply:
Please also use the `@apply` directe here to make use of the tailwind utilities we have 🙏

Reply:
> Please have a look at the colors that are used for the current gradients and reuse them, please

Now I've used the same colors.